src/modules/packet/Template.cpp

```diff
@@ -15,8 +15,6 @@
 #include <fstream>
 #include <iostream>
 
-#include "common/msg.h"
```

Comment:
I guess this change is not necessary, and the include can stay in the cpp file.

Reply:
You are right. I moved it because I was using the exception macro defined in the msg.h in the header, which is no longer the case.

Reply:
Please put it in the same position such that there is no diff visible.